BoostTestPlugin/source.extension.vsixmanifest

```diff
@@ -1,39 +1,39 @@
-    </Dependencies>
-    <Assets>
-        <Asset d:Source="Project" d:ProjectName="BoostTestAdapter" Type="Microsoft.VisualStudio.MefComponent" Path="|BoostTestAdapter|" />
-        <Asset d:Source="Project" d:ProjectName="BoostTestAdapter" Type="UnitTestExtension" Path="|BoostTestAdapter|" />
-        <Asset Type="Application Settings Configuration file" d:Source="File" Path="BoostTestAdapter.dll.config" />
-        <Asset Type="Microsoft.VisualStudio.Assembly" d:Source="Project" d:ProjectName="VisualStudioAdapter" Path="|VisualStudioAdapter|" AssemblyName="|VisualStudioAdapter;AssemblyName|" />
-        <Asset Type="Microsoft.VisualStudio.Assembly" d:Source="Project" d:ProjectName="Antlr.DOT" Path="|Antlr.DOT|" AssemblyName="|Antlr.DOT;AssemblyName|" />
-        <Asset Type="Microsoft.VisualStudio.Assembly" d:Source="Project" d:ProjectName="BoostTestShared" Path="|BoostTestShared|" AssemblyName="|BoostTestShared;AssemblyName|" />
-        <Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" d:ProjectName="BoostTestPackage" Path="|BoostTestPackage;PkgdefProjectOutputGroup|" />
-        <Asset Type="Microsoft.VisualStudio.ItemTemplate" d:Source="Project" d:ProjectName="BoostTestItemTemplate" d:TargetPath="|BoostTestItemTemplate;TemplateProjectOutputGroup|" Path="ItemTemplates" d:VsixSubPath="ItemTemplates" />
-    </Assets>
-    <Prerequisites>
-        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.8.27729.1,17.0)" DisplayName="Visual Studio core editor" />
-        <Prerequisite Id="Microsoft.VisualStudio.Component.VC.CoreIde" Version="[15.6.27406.0,17.0)" DisplayName="Visual Studio C++ core features" />
-    </Prerequisites>
-</PackageManifest>
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+    <Metadata>
+        <Identity Id="BoostUnitTestAdapter.95183667-1f65-419b-9cac-c9fbbed28f8b" Version="1.1.0.0" Language="en-US" Publisher="Microsoft" />
+        <DisplayName>Test Adapter for Boost.Test</DisplayName>
+        <Description xml:space="preserve">Enables Visual Studio's testing tools with unit tests written for Boost.Test.</Description>
+        <PackageId>Microsoft.VisualStudio.VC.Ide.TestAdapterForBoostTest</PackageId>
+        <MoreInfo>https://go.microsoft.com/fwlink/?linkid=853712</MoreInfo>
+        <License>use.txt</License>
+        <GettingStartedGuide>https://go.microsoft.com/fwlink/?linkid=853711</GettingStartedGuide>
+        <ReleaseNotes>Release Notes.txt</ReleaseNotes>
+        <Icon>Icons\Icon.ico</Icon>
+        <PreviewImage>Boost_preview.png</PreviewImage>
+        <Tags>C++, Unit Testing, Testing, test explorer, TDD, Boost, Boost Test Library, Visual Studio, Test Adapter</Tags>
+    </Metadata>
+    <Installation>
+        <InstallationTarget Version="[15.0,)" Id="Microsoft.VisualStudio.Pro" />

```

Comment:
Do these need to be updated to 17.0 or no?

Reply:
Tested it locally and this seems to still work if installed into the older versions. It's not recommended and wont happen through the installer, but it looks like we don't need to fully limit it.